package.json

```diff
@@ -81,9 +81,9 @@
     "marked": "^0.8.0",
     "preact": "10.15.1",
     "preact-custom-element": "^4.3.0",
-    "preact-iso": "2.2.0",
+    "preact-iso": "^2.6.2",
```

Comment:
`preact-iso` was locked as `onLoadEnd` (which we use for that progress-bar component) was never triggered w/out navigation. Refresh the page and you'd see the progress bar run forever

Fixed in `2.6.2` this morning though.